### ISSUE TITLE
`next_page` argument in stock price objects

### ISSUE THREAD

**Issue by Zhang-Haipeng** [UBC-MDS/pyntrinio#43]
So as discussed earlier, when the `start_date` and `end_date` are too far away, Intrinio will arbitrarily truncate the outputs to a maximum length of `100`.  
```
buy_date = '2017-01-01' 
sell_date = '2019-10-05'
intrinio_sdk.ApiClient().configuration.api_key['api_key'] = api_key
security_api = intrinio_sdk.SecurityApi()
api_response = security_api.get_security_stock_prices('AAPL', start_date=buy_date, end_date=sell_date)

api_response.stock_prices[-1].date, api_response.stock_prices[0].date

>>>(datetime.date(2019, 5, 15), datetime.date(2019, 10, 4))
```

The solution I've found so far, is to use `next_page` argument, like this: 
![image](https://user-images.githubusercontent.com/53522440/75845020-b1eb1680-5d8c-11ea-84dd-cd52a687dfde.png)

The `next_page` will update each time we call the object with the old `next_page`.

**Comment by Zhang-Haipeng** [comment]
Oh there's another way to do this and it's hopefully easier:  
```
while api_response.stock_prices[-1].date != buy_date:
  sell_date = api_response.stock_prices[-1].date
  api_response = security_api.get_security_stock_prices('AAPL', start_date=buy_date, end_date=sell_date)
  pd.append()
```

**Comment by slbsolomon** [comment]
Thanks for the update! will make updates to my code shortly

**Comment by slbsolomon** [comment]
What I have figured out:
- we can control this through the page_size argument in the stock prices call.
- the max value of the page_size is 10000, so I will make that the default to address this
- if a user inputs a start date or an end date, this is overridden 
- edge cases will be included in tests

**Comment by Zhang-Haipeng** [comment]
> What I have figured out:
> 
> * we can control this through the page_size argument in the stock prices call.
> * the max value of the page_size is 10000, so I will make that the default to address this
> * if a user inputs a start date or an end date, this is overridden
> * edge cases will be included in tests

Perfect, that would be ideal.  I'm not sure why it gave me error when I tried to set `page_size` to larger than 100. But as long as it's working on your end.

**Comment by slbsolomon** [comment]
In the limited access version, you can only ping the API with page size >100 once per minute, so that could be the problem

**Comment by slbsolomon** [comment]
Regardless, addressed in PR - the first draft of my tests are also completed

**Comment by slbsolomon** [comment]
I am leaving this issue as open in hopes to address an issue of over-pinging the API.

A few options:
1. Add an argument into the function to make page_size = 100 by default but allow it to be set to "max"
2. Add an if statement to calculate the number of days between start_date and datetime.today and if it is less than 100, only ping for 100

I may also do a combination of these 2

**Comment by slbsolomon** [comment]
Addressed in #56
